public/src/client/topic/events.js

```diff
@@ -130,10 +130,12 @@ define('forum/topic/events', [
 		};
 
 		templates.parse('partials/topic/post-editor', editData, function(html) {
-			translator.translate(html, function(translated) {
-				html = $(translated);
-				editorEl.replaceWith(html);
-				html.find('.timeago').timeago();
+			require(['translator'], function(translator) {
+				translator.translate(html, function(translated) {
+					html = $(translated);
+					editorEl.replaceWith(html);
+					html.find('.timeago').timeago();
+				});
```

Comment:
In this case, `translator` can be added to the very top, in the `require` line :smile: